hrm-domain/hrm-core/contact-job/contact-job-complete.ts

```diff
@@ -174,6 +177,21 @@ export const pollAndProcessCompletedContactJobs = async (jobMaxAttempts: number)
         const completedJob: CompletedContactJobBody = JSON.parse(m.Body);
 
         if (completedJob.attemptResult === ContactJobAttemptResult.SUCCESS) {
+          // I am not sure if I did this rught, but I am supposed to call the createContactJob and
+          // pass the additionalPayload
+          const contact = await getById(completedJob.accountSid, completedJob.contactId);
+          await createContactJob(completedJob)({
+            jobType: ContactJobType.SCRUB_CONTACT_TRANSCRIPT,
+            resource: contact,
+            additionalPayload: {
+              originalLocation: {
+                bucket: completedJob.filePath, // this is not right, how do I fetch the bucket url?,
+                key: completedJob.filePath, // this is not right, how do I fetch the key?
+              },
+            },
+          });
+
+          await scrubCompletedContactJobsFromQueue(completedJob); // Is this called the right way?
```

Comment:
This should be done as part of the pending jobs processor.

You should add the job to the table as pending here, like you are doing on the previous like, but you should post to the queue in the part of the code that picks up pending jobs from the table. There's code that already processes records with the retrieve transcript job type - you will need to add extra code to look for the scrub transcript job type and post it to the queue there